robottelo/ui/locators/common.py

```diff
@@ -75,7 +75,9 @@
 
     "search": (By.ID, "search"),
     "auto_search": (
-        By.XPATH, "//ul[contains(@id, 'ui-id')]/li/a[contains(., '%s')]"),
+        By.XPATH,
```

Comment:
I've checked manually if the new xpath is ok with organizations but haven't taken a look on location, will do that. Long story short previous xpath was working only with foreman-style search bar, new one should work with both foreman and katello styles.